widget.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Interactive graphics",
-  "package": "com.fliplet.interactive-map",
+  "name": "Interactive graphics-andrey",
+  "package": "com.fliplet.interactive-map-andrey",
```

Comment:
```suggestion
  "package": "com.fliplet.interactive-map",
```

Reply:
@tonytlwu Sorry this is my bad with this comment, lost this thing